diff -ru OrcaSlicer-2.3.0-beta.old/src/slic3r/Config/Version.cpp OrcaSlicer-2.3.0-beta/src/slic3r/Config/Version.cpp
--- OrcaSlicer-2.3.0-beta.old/src/slic3r/Config/Version.cpp	2025-02-22 12:58:23.000000000 +0300
+++ OrcaSlicer-2.3.0-beta/src/slic3r/Config/Version.cpp	2025-02-22 23:04:22.903067371 +0300
@@ -3,6 +3,7 @@
 #include <cctype>
 
 #include <boost/filesystem/operations.hpp>
+#include <boost/filesystem/directory.hpp>
 #include <boost/nowide/fstream.hpp>
 
 #include "libslic3r/libslic3r.h"
