diff -ru OrcaSlicer-2.3.0-beta.old/cmake/modules/Findlibnoise.cmake OrcaSlicer-2.3.0-beta/cmake/modules/Findlibnoise.cmake
--- OrcaSlicer-2.3.0-beta.old/cmake/modules/Findlibnoise.cmake	2025-02-22 12:58:23.000000000 +0300
+++ OrcaSlicer-2.3.0-beta/cmake/modules/Findlibnoise.cmake	2025-02-22 21:21:45.532505796 +0300
@@ -1,4 +1,4 @@
-find_path(LIBNOISE_INCLUDE_DIR libnoise/noise.h)
+find_path(LIBNOISE_INCLUDE_DIR noise/noise.h)
 find_library(LIBNOISE_LIBRARY NAMES libnoise libnoise_static liblibnoise_static)
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(libnoise DEFAULT_MSG
@@ -12,4 +12,4 @@
         IMPORTED_LOCATION "${LIBNOISE_LIBRARY}"
         INTERFACE_INCLUDE_DIRECTORIES "${LIBNOISE_INCLUDE_DIR}"
     )
-endif()
\ В конце файла нет новой строки
+endif()
diff -ru OrcaSlicer-2.3.0-beta.old/src/libslic3r/PerimeterGenerator.cpp OrcaSlicer-2.3.0-beta/src/libslic3r/PerimeterGenerator.cpp
--- OrcaSlicer-2.3.0-beta.old/src/libslic3r/PerimeterGenerator.cpp	2025-02-22 12:58:23.000000000 +0300
+++ OrcaSlicer-2.3.0-beta/src/libslic3r/PerimeterGenerator.cpp	2025-02-22 21:21:32.421846765 +0300
@@ -22,7 +22,7 @@
 #include "libslic3r/AABBTreeLines.hpp"
 #include "Print.hpp"
 #include "Algorithm/LineSplit.hpp"
-#include "libnoise/noise.h"
+#include "noise/noise.h"
 static const int overhang_sampling_number = 6;
 static const double narrow_loop_length_threshold = 10;
 static const double min_degree_gap = 0.1;
